src/styles/_catalog.scss

```
@@ -1,22 +1,38 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+
+  gap: 48px;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
 
   max-width: 944px;
-  margin: 0 auto;
+  margin: 0 auto 48px;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
```

Fix this in all places
```suggestion
  @media (min-width: 488px) 
      grid-template-columns: repeat(2, 200px);
  }
```